jekyll/_cci2/circleci-images.md

```diff
@@ -47,31 +56,35 @@ CircleCI maintains language images for the following languages:
 - [Python](#python)
 - [Ruby](#ruby)
 
-If your language is not listed, CircleCI also maintains a wizard you can use to create a custom image. See the [Dockerfile Wizard]({{ site.baseurl }}/2.0/custom-images/#circleci-dockerfile-wizard) section of the Using Custom-Built Docker Images document for instructions. 
+If your language is not listed,
+CircleCI also maintains a wizard you can use to create a custom image.
+See the [Dockerfile Wizard]({{ site.baseurl }}/2.0/custom-images/#circleci-dockerfile-wizard) section of the Using Custom-Built Docker Images document for instructions.
+
+**Note:**
+The above images are based on the **most current** upstream images for their respective languages.
+Because the latest images are more likely to change,
+it is best practice to use a more specific tag.
+For more details,
+see the [Customizing a Convenience Image](#customizing-a-convenience-image) section.
 
 ### Service Images
 
-Service images are images for services like databases. These images should be listed _after_ language images so they become secondary service containers.
+Service images are images for services like databases.
+These images should be listed **after** language images
+so they become secondary service containers.
 
-CircleCI maintains service images for the following services:
+CircleCI maintains images for the services below.
 
 - [buildpack-deps](#buildpack-deps)
 - [MariaDB](#mariadb)
 - [MongoDB](#mongodb)
 - [MySQL](#mysql)
 - [PostgreSQL](#postgresql)
 
-## How to Get Started with Pre-Built Docker Images Video Tutorial
-<div class="video-wrapper">
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/PgIwBzXBn7M" frameborder="0" allowfullscreen></iframe>
-</div>
-
 ## Pre-installed Tools
 
 All convenience images have been extended with additional tools.
 
-### APT Packages
-
 The following packages are installed with `apt-get` on every image:
```

Comment:
Maybe use `via` instead of `with` as it makes more contextual sense and matches what we use for curl right after.